packages/server/lib/modes/run.ts

```diff
@@ -676,7 +676,7 @@ async function waitForTestsToFinishRunning (options: { project: Project, screens
 
       span?.setAttributes({
         videoName,
-        videoCompression,
+        videoCompressionString: videoCompression.toString(),
```

Comment:
honeycomb decided this value was an int, so false === 0... i changed it to a string to get the real value.